Source/WebCore/rendering/RenderText.cpp

```diff
@@ -44,11 +44,13 @@
 #include "LineSelection.h"
 #include "LocalFrame.h"
 #include "LocalFrameView.h"
+#include "MathMLElement.h"
```

Comment:
It's kinda weird that RenderText depends on MathML elements for text transform conversion. I think that should be the other way around: RenderText would expose the API to do mathvariant conversion and MathML would use it.